darwin/cli_functions.py

```diff
@@ -5,8 +5,8 @@
 import sys
 from itertools import tee
 from pathlib import Path
-from typing import Dict, List, NoReturn, Optional, Union
-
+from typing import Callable, Dict, Iterator, List, NoReturn, Optional, Union, Tuple
+from darwin.dataset.release import Release
```

Comment:
I'm pretty sure this shouldn't belong here. Are you not automatically re-ordering import statements?